Tools/install.sh

```diff
@@ -22,6 +22,46 @@ FONT_PACKAGES=(\
     fonts-droid-fallback \
     )
 
+check_for_installed_tools () {
+    REQUIRED_TOOLS="${@}"
+    echo -e "Checking for installed tools.\nrequirded tools: ${REQUIRED_TOOLS}"
+    TOOLS_MISSING=0
+    for TOOL in ${REQUIRED_TOOLS}; do
+        if ! command -v "${TOOL}" > /dev/null ; then
+            TOOLS_MISSING=1
+            echo "Install required tool ${TOOL}"
+        fi
+    done
+    if [ $TOOLS_MISSING != 0 ]; then
+    echo "ABORTING"
+    if [ $sourced = 1 ]; then
+        return 1
+    else
+        exit 1
+    fi
+    fi
+}
+
+check_for_installed_files () {
+    REQUIRED_FILES="${@}"
```

Comment:
shellcheck: `SC2124 (warning): Assigning an array to a string! Assign as array, or use * instead of @ to concatenate.`